typescript_typeorm_graphql_project/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "node_project",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "node dist/app.js",
+    "dev": "ts-node src/app.ts --watch",
+    "build": "tsc",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@types/bcryptjs": "^2.4.6",
+    "@types/express": "^4.17.1",
+    "@types/jsonwebtoken": "^9.0.5",
+    "@types/node": "^20.10.7",
+    "typescript": "^5.3.3"
+  },
+  "dependencies": {
+    "@types/express-serve-static-core": "^4.17.41",
```

Comment:
What is the use of the lib?

Reply:
Removed this dependency as it is currently not being used.